lib/omnibus/packagers/rpm.rb

```diff
@@ -630,8 +630,27 @@ def install(packages, enablerepo = NULL)
         log.info(log_key) do
           'enablerepo only works on yum based systems, not on zypper based ones'
         end
-        shellout!('zypper clean')
-        shellout!("zypper install -y #{packages}")
+        log.info(log_key) do
+          'zypper ms -d -a'
```

Comment:
it disables the service so that we get fewer failures when it refreshes the repos